app/src/behaviors/behavior_sleep.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#define DT_DRV_COMPAT zmk_behavior_sleep
+
+#include <device.h>
+#include <drivers/behavior.h>
+#include <logging/log.h>
+
+#include <zmk/behavior.h>
+#include <zmk/activity.h>
+
+#if DT_HAS_COMPAT_STATUS_OKAY(DT_DRV_COMPAT)
+
+LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
+
+static int behavior_sleep_init(const struct device *dev) { return 0; };
+
+static int on_keymap_binding_released(struct zmk_behavior_binding *binding,
+                                      struct zmk_behavior_binding_event event) {
+#if IS_ENABLED(CONFIG_ZMK_SLEEP)
+    return activity_set_state(ZMK_ACTIVITY_SLEEP);
+#endif
+    return ZMK_BEHAVIOR_OPAQUE;
+}
+
+static int on_keymap_binding_pressed(struct zmk_behavior_binding *binding,
+                                     struct zmk_behavior_binding_event event) {
+    return ZMK_BEHAVIOR_OPAQUE;
+}
+
+static const struct behavior_driver_api behavior_sleep_driver_api = {
+    .binding_released = on_keymap_binding_released,
+    .binding_pressed = on_keymap_binding_pressed,
```

Comment:
This should have a `.locality` set:

```suggestion
    .binding_pressed = on_keymap_binding_pressed,
    .locality = BEHAVIOR_LOCALITY_GLOBAL,
```